### ISSUE TITLE
Feature Request: Custom Time Format to Start/Received/Due/End and Custom Field card/minicard

### ISSUE THREAD

**Issue by xxfelix** [wekan/wekan#2011]
The Due and Start Date time on card is currently in HH:mm A format. This is not always convenient if I want to enter time.

Is it convenient we can use 24h instead 12h, also if I change the language, if I change to chinese, I need to input "中午12点00"， it's also not friendly to chinese.

So we all accept international time, we don't need to translate the time to local language, maybe just use 24h like "16:15" is the most convenient no matter what language it is, what's your idea?

**Comment by nightwind27** [comment]
yes   this is a very important things, each time we select time, it can't be saved.
please make a drop list only have 24hours, that is best for team to use

**Comment by nightwind27** [comment]
**this is a very important things, each time we select time, it can't be saved.
please make a drop list only have 24hours, that is best for team to use**

**Comment by nightwind27** [comment]
now , the date can be selected, but time only support input, in the work, time is not nessary hh:mm, only show hh (be selected is better than input )
